POSTGRES.md

```diff
@@ -0,0 +1,56 @@
+<img src="./images/daml-enterprise-logo.svg" width="400px">
+
+[![Artifact Hub](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/digital-asset)](https://artifacthub.io/packages/search?repo=digital-asset)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](./LICENSE)
+
+# Postgresql database requirements
+
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Versions](#versions)
+- [Network connection](#network-connection)
+- [License](#license)
+
+## Introduction
+
+This document intends to describe the required postgres database setup as dependency for daml-helm-charts deployment.
+
+## Versions
+
+The following postresql major versions are tested and supported by our components:
+- 11.x
+- 12.x
+- 13.x
+- 14.x
+
+## Network connection
+
+[Max connections](https://www.postgresql.org/docs/14/runtime-config-connection.html#GUC-MAX-CONNECTIONS) setting of Postgresql must be set according to the number of components installed.
+A minimal reference installation using the default values in this repository in all helm charts requires at least 140 connections. 
+
+The deployed components must have network connection to the database instance(s).
+Connecting to the database is described in our helm charts README files.
+
+For example:
+[Canton domain](https://github.com/digital-asset/daml-helm-charts/tree/main/charts/canton-domain#minimum-viable-configuration)
+Under storage values.
+
+## [Contributing guidelines](./CONTRIBUTING.md)
```

Comment:
No need for contributing and licence here IMO

Reply:
Good point, removed